components/02-molecules/pull-quote/_pull-quote.scss

```diff
@@ -0,0 +1,59 @@
+:root {
+  --color-gray-light: #{$gray-light};
+  --color-gray-lightest: #{$gray-lightest};
+  --color-blue-light: #{$blue-light};
+  --color-black: #{$black};
+}
+
+.pull-quote {
+  --color-pull-quote-quote: var(--color-gray-lightest);
+  --color-pull-quote-attribution: var(--color-gray-lightest);
+  --color-pull-quote-accent: var(--color-blue-light);
+
+  background-color: var(--color-black);
+  padding: 3rem 2rem;
+
+  + .pull-quote {
+    margin-top: 2rem;
+  }
+}
+
+.pull-quote__figure {
+  [data-pull-quote-style='bar-left'] & {
+    padding-left: $space;
+    border-left: 0.5rem solid var(--color-pull-quote-accent);
```

Comment:
See my color comment below. Any instance of a custom property should use the clr function instead. `clr(accent)` <-- this function returns a custom property but also allows us to implement theme's (currently just detecting OS darkmode) and redefine color values based on the active theme.